src/backend/adapters/SecretFileAdapter.ts

```diff
@@ -0,0 +1,45 @@
+import { randomBytes } from "crypto"
+import { SecretManagerAdapter } from "../services/UserService"
+import { FileStorageHelper } from "../utils/fileStorageHelper"
+import { ErrorType } from "../utils/tryCatch"
+
+const secretFileName = "secret"
+
+export const makeSecretFileAdapter = (
```

Comment:
the secret adapter is a guy that can produce a secret (which is also is storing on disk).  This secret is used to symmetrically encrypt jwt's.  It's generated programmatically if it doesn't exist.

I could have also just added the secret as an environment variable but I like doing it programmatically to reduce config.  Also I could make the secrets rotate or something with programmaticly generated secrets.